cmd/bootkube/render.go

```diff
@@ -50,6 +50,8 @@ var (
 		serviceCIDR         string
 		selfHostKubelet     bool
 		cloudProvider       string
+		networkProvider     string
+		networkMTU          int
```

Comment:
Can we get away with a sane default here rather than a flag? I'd really like to avoid this

Reply:
Agreed, I was on the fence about this. An appropriate default is 1440, but folks really do need to pick the right value depending on the cloud provider or network performance suffers. Since `bootkube render` just produces valid example manifests (not ideal ones), perhaps we leave it to projects to generate assets that use the correct MTU. In my cases, I generate the right assets for the platform separately, so I'm not opinionated which way `bootkube render` goes.

Reply:
Yeah?

Reply:
Removed. flannel and Calico use different defaults, so setting it in one place breaks tests anyway. This means `bootkube render` will create clusters that work using the default settings, but aren't necessarily getting optimal throughput. I think that's ok given that higher level tools do handle these cases.